setup.py

```diff
@@ -88,7 +88,7 @@
         "numpy",
         "orix           >= 0.9",
         "psutil",
-        "pyfai",
+        "pyfai          <=2023.1.0",  # sigma clip function broken
```

Comment:
Is the broken function tracked anywhere? Would be good to refer to it here, so that we know when to update the version, excluding 2023.2.0 and any other versions where it is broken.

Reply:
I guess after looking more into this the function isn't broken they just changed the argument from npt_rad to npt.  Honestly the sigma_clip function should probably be removed as it isn't used that much. 

I don't love the pyfai dependency as they do have a tendency to do things like this.  Doing version checking is also difficult as they don't follow standards very well.

Reply:
So updating pyxem to use the parameter `npt` instead of `npt_rad` and set the minimal version to 2023.2.0 works?

Reply:
Yea I added in the version checking to the function which fixes the problem and allows you to use 2023.2.0